Hallo.Giraffe.Sample/Startup.fs

```diff
@@ -0,0 +1,78 @@
+namespace Hallo.Giraffe.Sample
+
+open System.Text.Encodings.Web
+open System.Text.Json
+open System.Text.Json.Serialization
+open Hallo
+open Hallo.AspNetCore
+open Hallo.AspNetCore.Sample
+open Hallo.AspNetCore.Sample.Data
+open Hallo.AspNetCore.Sample.Models
+open Hallo.AspNetCore.Sample.Models.Hypermedia
+open Microsoft.AspNetCore.Builder
+open Microsoft.AspNetCore.Hosting
+open Microsoft.AspNetCore.Http
+open Microsoft.Extensions.DependencyInjection
+open Giraffe
+open FSharp.Control.Tasks
+
+type Startup() =
+
+    let getPeopleHandler: HttpHandler =
+        fun (next: HttpFunc) (ctx: HttpContext) ->
+            task {
+                let peopleRepo =
+                    ctx.RequestServices.GetRequiredService<PeopleRepository>()
+
+                let model = peopleRepo.List(Paging())
+                do! HalJsonGenerator.HalHandler(ctx, model)
+                return Some ctx
+            }
+
+    let getPeopleByIdHandler (id: int): HttpHandler =
+        fun (next: HttpFunc) (ctx: HttpContext) ->
+            task {
+                let peopleRepo =
+                    ctx.RequestServices.GetRequiredService<PeopleRepository>()
+
+                let model = peopleRepo.Get(id)
+                do! HalJsonGenerator.HalHandler(ctx, model)
+                return Some ctx
+            }
+
+
+    let webApp =
+        choose [ route "/"
+                 >=> GET
+                 >=> text "There's no place like 127.0.0.1" //TODO Home Document application/json-home?
+
+                 route "/people" >=> GET >=> getPeopleHandler
+                 GET >=> routef "/people/%i" getPeopleByIdHandler
+
+                  ]
+
+    member _.ConfigureServices(services: IServiceCollection) =
+        services.AddSingleton<PeopleRepository>()
```

Comment:
For ease of use, F# sample references the ASP.NET Core sample app to use the same types and representations